src/components/PokemonPage.jsx

```diff
@@ -82,6 +82,14 @@ const PokemonPage = () => {
     }
   };
 
+  const handleEventNext = () => {
+    history.push(`/id/${infos.id + 1}`);
+  };
+
+  const handleEventPrevious = () => {
+    history.push(`/id/${infos.id - 1}`);
```

Comment:
Beware of edge cases: what happens if I'm on Bulbasaur (#1) ? Or on the last one ?